Sources/PackageGraph/Pubgrub.swift

```diff
@@ -751,7 +836,9 @@ public final class PubgrubDependencyResolver<
             //            if case .satisfied = solution.satisfies(candidateIncompat) {
             //                continue // TODO: is this correct?
             //            }
-            solution.decide(Term(candidate.package, .versionSet(.exact(version))))
+            let decision = Term(candidate.package, .versionSet(.exact(version)))
+            solution.decide(decision)
```

Comment:
Maybe take the decide API take the package reference and the decided version so the client doesn't have to create the .exact requirement.